### DISCUSSION TITLE
KeyError: 'Non-existent config key: USE_LAYOUT'

### DISCUSSION THREAD

**Discussion by himasai9711** [deepdoctection/deepdoctection#192] [Q&A]
when i changed my config file path , im getting this error

<img width="1313" alt="image" src="https://github.com/deepdoctection/deepdoctection/assets/75134122/44196210-708d-4cca-9da8-45d423c23a20">

**Comment by JaMe76** [comment]
You are using config of deepdoctection  analyzer but you must use config of a detectron2 model

**Reply by himasai9711** [reply]
where can i get that?
